### PR TITLE
generate docs to show concise data if user so asks issue #723

### PR DESCRIPTION
**Pull request by vikasgupta78**
generate docs to show concise data if user so asks issue #723

### CODE REVIEW

File: common/client/src/main/java/zingg/common/client/Arguments.java [hunk 1]
```diff
@@ -169,7 +170,21 @@ public void setLabelDataSampleSize(float labelDataSampleSize) throws ZinggClient
 	public List<? extends FieldDefinition> getFieldDefinition() {
 		return fieldDefinition;
 	}
-
+	
+	@JsonIgnore @Override
```
Comment (sonalgoyal) [inline common/client/src/main/java/zingg/common/client/Arguments.java:174]: we should move this to a separate class. args dont need to know about match types

**Review by sonalgoyal — CHANGES_REQUESTED** [review]

File: common/client/src/main/java/zingg/common/client/Arguments.java (hunk 1 above)
Comment (vikasgupta78) [reply common/client/src/main/java/zingg/common/client/Arguments.java:174]: What would be a good place? I could have kept it in ModelDocumenter but keeping it at common place might be useful to others in future.

Comment (sonalgoyal) [reply common/client/src/main/java/zingg/common/client/Arguments.java:174]: yes it is likely useful to others. I think a FieldDefUtil may be ok for now.

Comment (vikasgupta78) [reply common/client/src/main/java/zingg/common/client/Arguments.java]: Done in latest commit

File: common/client/src/test/java/zingg/common/client/TestArguments.java [hunk 2]
```diff
@@ -25,6 +26,8 @@ public class TestArguments {
 
 	public static final Log LOG = LogFactory.getLog(TestArguments.class);
 	protected ArgumentsUtil argsUtil = new ArgumentsUtil();
+	
+	protected FieldDefUtil fieldDefUtil = new FieldDefUtil();
```
Comment (sonalgoyal) [inline common/client/src/test/java/zingg/common/client/TestArguments.java:30]: These tests should be moved to a separate class for the util.

**Review by sonalgoyal — CHANGES_REQUESTED** [review]
Junit changes

File: common/client/src/test/java/zingg/common/client/TestArguments.java (hunk 2 above)
Comment (vikasgupta78) [reply common/client/src/test/java/zingg/common/client/TestArguments.java:30]: Done in latest commit

**Pull request merged**